WalletWasabi/Crypto/ZeroKnowledge/ProofSystem.cs

```diff
@@ -11,7 +12,7 @@ public static class ProofSystem
 	{
 		private static GroupElement O = GroupElement.Infinity;
 
-		public static bool Verify(LinearRelation.Statement statement, Proof proof)
+		public static bool Verify(Statement statement, Proof proof)
 		{
 			return Verifier.Verify(new Transcript(new byte[0]), new[] { statement }, new[] { proof });
```

Comment:
and then this:

```suggestion
			return Verifier.Verify(new Transcript(label: EmptyLabel), new[] { statement }, new[] { proof });
```

(also in Prover but github doesn't want to let me add a suggestion there)